virt/virt-2-4-release-notes.adoc

```diff
@@ -195,5 +192,10 @@ nodes.
 running either `oc adm drain` or `kubectl drain`. Do not run these commands on
 the nodes of any clusters where {VirtProductName} is deployed. The nodes might not
 drain if there are virtual machines running on top of them.
+<<<<<<< HEAD
```

Comment:
@lmandavi - this hasn't been cleanly fixed.

Reply:
@vikram-redhat @kalexand-rh I am looking at this again.

Reply:
@lmandavi Should we just close this PR? The release note it was adding has been nixed. Functionally now it is only removing some placeholder comments (once the conflict is resolved)

Reply:
+1 to just closing the PR.

Reply:
OK. Closing this PR.